scripts/build.mjs

```diff
@@ -56,14 +59,19 @@ parameters
     .option("--override-css <main css file>", "pass in an alternative main css file")
 parameters.parse(process.argv);
 
+function srcPath(src) {
```

Comment:
can we name this `modulePath` or something referring more the fact that this is only for javascript files used as an es module (e.g. not the service worker atm). Also, can you add a comment saying as much?

Reply:
What about these lines of code over here? Seems like the service worker is also included via `srcPath`.

https://github.com/vector-im/hydrogen-web/blob/b6d0d5424191bae9d7df2acb52062d6fbf022380/scripts/build.mjs#L109-L110

Should this not be the case?

Reply:
Have discussed OOB, this doesn't matter too much either way, until we want to use TS in the sw, so is fine to stay as is.